### PR TITLE
cleanup: ECDSAContractAccount

### PR DESCRIPTION
**Pull request by kfichter**
## Description
More minor cleanups.

## Contributing Agreement

- [x] I have read and understood the [Optimism Contributing Guide and Code of Conduct](https://github.com/ethereum-optimism/optimism-monorepo/blob/master/.github/CONTRIBUTING.md) and am following those guidelines in this pull request.

### CODE REVIEW

File: contracts/optimistic-ethereum/OVM/accounts/OVM_ECDSAContractAccount.sol [hunk 1]
```diff
@@ -22,8 +22,15 @@ import { Lib_SafeMathWrapper } from "../../libraries/wrappers/Lib_SafeMathWrappe
  */
 contract OVM_ECDSAContractAccount is iOVM_ECDSAContractAccount {
 
+    /*************
+     * Constants *
+     *************/
+
+    // TODO: should be the amount sufficient to cover the gas costs of all of the transactions up
```
Comment (tynes) [inline contracts/optimistic-ethereum/OVM/accounts/OVM_ECDSAContractAccount.sol:29]: Is this still a `TODO`?

Comment (tynes) [reply contracts/optimistic-ethereum/OVM/accounts/OVM_ECDSAContractAccount.sol:29]: It seems more like a note

Comment (ben-chain) [reply contracts/optimistic-ethereum/OVM/accounts/OVM_ECDSAContractAccount.sol:29]: I agree it's more of a note; we now have a ticket to update intrinsic gas so I think fine to leave as is and update when we do that.

**Review by ben-chain — APPROVED** [review]
👍

**Pull request merged**